src/stock-items/add-stock-item/stock-item-rules/add-stock-rules.component.tsx

```diff
@@ -60,9 +66,13 @@ const StockRulesAddOrUpdate: React.FC<AddStockRuleProps> = ({
   const { items: dispensingUnits } = useStockItemPackagingUOMs(stockItemFilter);
 
   //locations
-  const {
-    locations: { results: locations },
-  } = useStockLocations({ v: ResourceRepresentation.Default });
+  // const {
+  //   locations: { results: locations },
+  // } = useStockLocations({ v: ResourceRepresentation.Default });
+
```

Comment:
Remove the commented out code

Reply:
this has been removed in the clean commit